src/System.Collections/src/System/Collections/BitArray.cs

```diff
@@ -341,6 +341,128 @@ public BitArray Not()
             return this;
         }
 
+        /*=========================================================================
+        ** Shift all the bit values to right on count bits. When count less zero it
+        ** shift all the bit values to left. New bits are be set to zero. The 
+        ** current instance is updated and returned.
+        =========================================================================*/
```

Comment:
It seems weird and a bit confusing to me that you can either invoke LeftShift(n) or RightShift(-n) and get the same behavior. I'd think either negative values would not be valid, or there would be a single Shift method that accepts negative or positive values. Ultimately though, to me it makes the most sense to just match the behavior of the primitive shift operators (which this currently does not).